### PR TITLE
Let expectations return/raise values

### LINKED ISSUE #78: Let expect return a value
This has come up a lot of times so I'm going to add it back in.

### PR DESCRIPTION
**Pull request by toddsifleet**
* Fixes #78

### CODE REVIEW

**Comment by toddsifleet** [comment]
@charlax @lucasmarshall @breerly What do you guys think of this?

File: test/allow_test.py [hunk 1]
```diff
@@ -75,48 +75,49 @@ def test_proxies_name(self):
         assert subject.method_with_doc.__name__ == "method_with_doc"
 
 
+@mark.parametrize('stubber', [allow, expect])
```
Comment (dnathe4th) [inline test/allow_test.py]: Kind of weird to be testing `expect` inside the `allow_test.py` file.

**Comment by dnathe4th** [comment]
I approve of this change

**Comment by toddsifleet** [comment]
I agree, maybe I should pull it out into stub_return_test.py

**Comment by toddsifleet** [comment]
Merged

**Pull request closed without merging**